test/integration/pool.test.js

```diff
@@ -16,7 +16,9 @@ function assertSameConnection(newConnection, oldConnection) {
     case 'postgres':
       expect(oldConnection.processID).to.be.equal(newConnection.processID).and.to.be.ok;
       break;
-
+    case 'yugabytedb':
```

Comment:
you can deduplicate code by doing it the same way as mariadb&mysql instead:

![image](https://user-images.githubusercontent.com/1280915/162497588-2430fe13-61c2-4209-89a5-7cf98db36392.png)